src/components/create-post.jsx

```diff
@@ -190,7 +205,7 @@ export default class CreatePost extends React.Component {
 
           <button className="btn btn-default btn-xs"
             onClick={preventDefault(this.createPost)}
-            disabled={this.state.isFormEmpty || this.props.createPostViewState.isPending}>Post</button>
+            disabled={this.state.isFormEmpty || this.state.attachmentQueueLength || this.props.createPostViewState.isPending}>Post</button>
```

Comment:
this.state.attachmentQueueLength is not a boolean, correct check would be `this.state.attachmentQueueLength > 0`